src/commands/fun/diceroll.ts

```diff
@@ -28,29 +33,58 @@ export default class extends Command {
 	run(_msg: Message): Promise<void> { return; }
 
 	tempRun(interaction: CommandInteraction): Promise<void> {
-		const min = interaction.options.getNumber('minimum') || DEFAULT_RANGE[0];
-		const max = interaction.options.getNumber('maximum') || DEFAULT_RANGE[1];
-		return interaction.reply(`Your random number is ${Math.floor((Math.random() * (max - min + 1)) + min)}`);
-	}
+		let min = interaction.options.getNumber('minimum');
+		let max = interaction.options.getNumber('maximum');
+		const numRolls = interaction.options.getNumber('numdice') || DEFAULT_ROLLS;
 
-	argParser(_msg: Message, input: string): Array<number> {
-		const [minimum, maximum, ...rest] = input.split('|').map(arg => arg.trim());
-		if (!input) {
-			return DEFAULT_RANGE;
+		let responseEmbed: MessageEmbed;
+		if (!min) {
+			[min, max] = [DEFAULT_RANGE[0], DEFAULT_RANGE[1]];
+		} else if (!max && min) {
+			responseEmbed = new MessageEmbed()
+				.setColor('#ff0000')
+				.setTitle('Argument error')
+				.setDescription('If you provide a minimum, you must also provide a maximum.');
+			return interaction.reply({ embeds: [responseEmbed], ephemeral: true });
```

Comment:
Same deal as with the argument errors in the role command. You might want to make that function a util.

Reply:
Done, and also integrated into the other commands that use this :)